homeassistant/components/modbus/climate.py

```diff
@@ -209,7 +209,7 @@ def target_temperature_step(self):
     def set_temperature(self, **kwargs):
         """Set new target temperature."""
         target_temperature = int(
-            (kwargs.get(ATTR_TEMPERATURE) - self._offset) / self._scale
+            (kwargs.get(CONF_TEMPERATURE) - self._offset) / self._scale
```

Comment:
Please revert this change.

Reply:
This is near online :-) I was preparing the commit as you requested the change :-)